web/src/i18n/locales/en.json

```
@@ -37,7 +37,7 @@
         "qualifyA": "The biggest hurdle to public speaking is often our own fears. We welcome any woman with an interest in public speaking to nominate herself, regardless of training or experience.",
         "qualifyQ": "Are there prerequisites for being a speaker?",
         "title": "Frequently Asked Questions",
-        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
+        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer, and Ania Nakayama. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
```

```suggestion
        "whoA": "This is an open source project created by Yan Fan, Tutti Quintella, Ann Kilzer. Current maintainers are Ann Kilzer and Ania Nakayama. The original idea comes from Emi Takemura Miller. Additional assistance was provided by Women Who Code Tokyo. If you’d like to make a contribution, please check out our GitHub page.",
```

Sorry for the back and forth... is this okay? Might be a bit more accurate 😇

---

And we might want to put this in the Japanese version too

---

@ann-kilzer added!